### DISCUSSION TITLE
Nowy sensor

### DISCUSSION THREAD

**Discussion by Raddii** [PiotrMachowski/Home-Assistant-custom-components-Burze.dzis.net#16] [Q&A] [answered]
Cześć, na burze.dzis.net pojawił się nowy sensor w związku z sytuacja na Ukrainie tzn. Promieniowanie gamma w Polsce. Pytanie czy zamierzasz dodać ten czujnik do integracji?

**Comment by PiotrMachowski** [comment]
Cześć, muszę sprawdzić, czy jest dostępny w API

**Reply by PiotrMachowski** [reply]
Dane nie są udostępniane w API Burze.dziś.net, najlepiej by było zrobić dedykowany sensor pobierający dane bezpośrednio ze źródła (https://www.gov.pl/web/paa/sytuacja-radiacyjna). Ewentualnie widziałem też scrape'owanie z HTMLa: https://pastebin.com/NTcJJZQF

**Reply by pkar70** [reply]
Jest też http://radioactiveathome.org

**Comment by PiotrMachowski** [comment, marked as answer]
Nowe wartości pojawiły się w API i zostały dodane w wersji `v2.0.0`
